code-scanning/properties/credo.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Credo Scan",
+    "creator": "Credo",
+    "description": "Credo is a static code analysis tool for the Elixir language with a focus on teaching and code consistency.",
+    "categories": ["Code Scanning", "Elixir"]
+}
```

Comment:
Can we add a newline to end of this file?

Reply:
Hi @jtamsut , I just added a newline but it's not appearing 🤔 looks like other files are similar. Let me know if there are still any issues here 👍🏻 I'm happy to try and fix again.